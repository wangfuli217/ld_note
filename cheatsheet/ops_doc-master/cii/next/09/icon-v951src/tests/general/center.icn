#SRC: JCON

procedure main()
    write(center("12345",3, ""))
    write(center("1",3, ""))
    write(center("123"))
    write(center("12"))
    write(center("1234",1))
    write(center("1234",2))
    write(center("1234",3))
    write(center("12345",1))
    write(center("12345",2))
    write(center("12345",3))
    write(center("123",8))
    write(center("12",8))
    write(center("123",9))
    write(center("12",9))
    write(center("123",8,"<>"))
    write(center("12",8,"<>"))
    write(center("123",9,"<>"))
    write(center("12",9,"<>"))
end
