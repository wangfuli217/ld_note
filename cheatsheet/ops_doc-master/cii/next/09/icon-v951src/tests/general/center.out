234
 1 
2
2
3
23
234
3
34
234
  123   
   12   
   123   
   12    
<>123><>
<><12><>
<><123><>
<><12<><>
